app/views/pages/home.html.erb

```diff
@@ -1,2 +1,2 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<h2>This is our amazing app!</h2>
```

Comment:
I would advise to change the `h2` to a `h1` for SEO purposes!

Reply:
Oh man, missed that one. Let me correct.